src/TestableIO.System.IO.Abstractions/IDirectoryInfoFactory.cs

```diff
@@ -4,7 +4,7 @@ namespace System.IO.Abstractions
     public interface IDirectoryInfoFactory
     {
         /// <summary>
-        /// Initializes a new instance of the <see cref="DirectoryInfoBase"/> class, which acts as a wrapper for a directory path.
+        /// Initializes a new instance of the <see cref="IDirectoryInfo"/> class, which acts as a wrapper for a directory path.
```

Comment:
Factually incorrect, since `IDirectoryInfo` is not a class? Also, do we actually need to say that it's a wrapper for something, rather than "which provides access to a directory" or similar?

Reply:
I only changed the `cref` attribute, so that it compiles. I think it would make sense to check all comments especially in the interface project, but would suggest to do this separately. If I update them now, GIT will lose the rename-reference and we might lose the history of the files...
What do you think?

Reply:
I agree, let's keep the content changes minimal and revisit afterwards.